psga/sars_cov_2/ncov2019_artic.nf

```diff
@@ -176,6 +190,10 @@ process ncov2019_artic_nf_pipeline_medaka {
       mv ${file_to_update} ${file_to_update%.*}.fa
   done
 
+  for file_to_update in `ls ${output_typing}/${ncov_prefix}_${sample_id}*.*`; do
```

Comment:
OPTIONAL CHANGE:
Recommend a comment at the top of this block explaining why it was done. (Like the update sample name)